src/stream.jl

```diff
@@ -149,7 +139,7 @@ function new_sequence_token(stream::CloudWatchLogStream)
     return new_sequence_token(stream.config, stream.log_group_name, stream.log_stream_name)
 end
 
-describe_log_streams(config; kwargs...) = logs(config, "DescribeLogStreams"; kwargs...)
+describe_log_streams(config::AWSConfig, log_group_name::AbstractString, params::AbstractDict) = CloudWatch_Logs.describe_log_streams(log_group_name, params; aws_config=config)
```

Comment:
Going to need to split this up significantly for line length restrictions

Reply:
I was going to create a follow up PR afterwards to just format the entire repo in BlueStyle